code/modules/events/shuttle_catastrophe.dm

```diff
@@ -19,6 +19,14 @@
 		return FALSE //don't remove all players when its already on station or going to centcom
 	return TRUE
 
+/datum/round_event_control/shuttle_catastrophe/admin_setup(mob/admin)
+	. = ..()
+
+	if(EMERGENCY_AT_LEAST_DOCKED || istype(SSshuttle.emergency, /obj/docking_port/mobile/emergency/shuttle_build))
+		if(tgui_alert(usr, "WARNING: This will unload the currently docked emergency shuttle and anything on it. Proceed anyways?", "How about a REAL catastrophe?", list("Yes", "No")) == "Yes")
```

Comment:
unloading implies emptying everyone off it, while true, doesn't convey the tragedy in the statement. how about "erase"?